src/WebJobs.Extensions.ApiHub/Listener/ApiHubListener.cs

```diff
@@ -51,31 +69,67 @@ public void Dispose()
             // nop
         }
 
-        public Task StartAsync(CancellationToken cancellationToken)
+        public async Task StartAsync(CancellationToken cancellationToken)
         {
-            _poll = _folderSource.CreateFileWatcher(_fileWatcherType, OnFileWatcher, nextItem: null, pollIntervalInSeconds: _pollIntervalInSeconds > 0 ? _pollIntervalInSeconds : DefaultPollIntervalInSeconds);
+            CloudStorageAccount account;
+            if (!CloudStorageAccount.TryParse(this._config.StorageConnectionString, out account))
```

Comment:
You don't need this validation. If this connection was invalid, the core SDK startup would have errored out much earlier